toolkit/quickrebuild_autoconfig.mk

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+# Contains:
+#	- build optimization flag auto configuration
+
+######## QUICKREBUILD AUTO CONFIGURE ########
+
+# The QUICKREBUILD* flags are special flags that will try to build the toolchain and packages as quickly as possible. They will
```

Comment:
Change leak from #5199?

Reply:
Yes, makes my life easier to validating the commands. Will clean up before the final merge.